vms/proposervm/batched_vm_test.go

```diff
@@ -225,7 +225,7 @@ func TestGetAncestorsPostForkOnly(t *testing.T) {
 	// prepare build of next block
 	require.NoError(builtBlk1.Verify(context.Background()))
 	require.NoError(proRemoteVM.SetPreference(context.Background(), builtBlk1.ID()))
-	proRemoteVM.Set(proRemoteVM.Time().Add(proposer.MaxDelay))
+	proRemoteVM.Set(proRemoteVM.Time().Add(proposer.MaxVerifyDelay))
```

Comment:
The reason this works is so weird. It just so happens that the `buildDelay` for this validator is equal to `MaxVerifyDelay`. Would it make more sense to specify this as `MaxBuildDelay` so that we are guaranteed for the following build block to work correctly? I haven't looked into other tests but my guess is that they will be similar.